src/core/AutoRest.Core/FileTemplate.cs

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
I thought we didn't check in the generated `.cs` files for `.cshtml` files since they are generated at build time?